sdk/policyinsights/Azure.ResourceManager.PolicyInsights/api/Azure.ResourceManager.PolicyInsights.netstandard2.0.cs

```diff
@@ -69,26 +75,34 @@ public static partial class PolicyInsightsExtensions
         public static Azure.AsyncPageable<Azure.ResourceManager.PolicyInsights.Models.PolicyTrackedResourceRecord> GetPolicyTrackedResourceQueryResultsAsync(this Azure.ResourceManager.Resources.SubscriptionResource subscriptionResource, Azure.ResourceManager.PolicyInsights.Models.PolicyTrackedResourceType policyTrackedResourcesResource, Azure.ResourceManager.PolicyInsights.Models.PolicyQuerySettings queryOptions = null, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
-        public static Azure.ResourceManager.PolicyInsights.ResourceGroupPolicyAssignmentPolicyInsightsResource GetResourceGroupPolicyAssignmentPolicyInsightsResource(this Azure.ResourceManager.ArmClient client, Azure.Core.ResourceIdentifier id) { throw null; }
-        public static Azure.ResourceManager.PolicyInsights.SubscriptionPolicyAssignmentPolicyInsightsResource GetSubscriptionPolicyAssignmentPolicyInsightsResource(this Azure.ResourceManager.ArmClient client, Azure.Core.ResourceIdentifier id) { throw null; }
-        public static Azure.ResourceManager.PolicyInsights.SubscriptionPolicyDefinitionPolicyInsightsResource GetSubscriptionPolicyDefinitionPolicyInsightsResource(this Azure.ResourceManager.ArmClient client, Azure.Core.ResourceIdentifier id) { throw null; }
-        public static Azure.ResourceManager.PolicyInsights.SubscriptionPolicySetDefinitionPolicyInsightsResource GetSubscriptionPolicySetDefinitionPolicyInsightsResource(this Azure.ResourceManager.ArmClient client, Azure.Core.ResourceIdentifier id) { throw null; }
         public static Azure.Pageable<Azure.ResourceManager.PolicyInsights.Models.PolicySummary> SummarizePolicyStates(this Azure.ResourceManager.ManagementGroups.ManagementGroupResource managementGroupResource, Azure.ResourceManager.PolicyInsights.Models.PolicyStateSummaryType policyStatesSummaryResource, Azure.ResourceManager.PolicyInsights.Models.PolicyQuerySettings queryOptions = null, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        public static Azure.Pageable<Azure.ResourceManager.PolicyInsights.Models.PolicySummary> SummarizePolicyStates(this Azure.ResourceManager.Resources.PolicyAssignmentResource policyAssignmentResource, Azure.ResourceManager.PolicyInsights.Models.PolicyStateSummaryType policyStatesSummaryResource, Azure.ResourceManager.PolicyInsights.Models.PolicyQuerySettings queryOptions = null, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
```

Comment:
Same here, we should rename parameter `policyStatesSummaryResource` to `policyStateSummaryType`